src/main/scala/ArgsParser.scala

```diff
@@ -105,6 +106,11 @@ class ArgsParser {
     opt[Int]("part-cutoff").action( (x, c) => c.copy(
         partCutoff = x)
     ).text("parameter used for partitioning")
+
+    opt[Unit]("vcd-on").action( (_, c) => c.copy(
```

Comment:
perhaps withVCD is a more clear flag than vcd-on (which might imply it can be turned on and off mid-simulation)

Reply:
Added withVCD flag with the recent commit